src/low-preview/ci.service.js

```diff
@@ -154,6 +154,33 @@ export default class CIResponsive {
       { isUpdate, imgNode, ratio, lazy, placeholderBackground, preserveSize, isPreview, ...imgProps }
     );
 
+    const observer = new MutationObserver(mutations => {
+      mutations.forEach(record => {
+        if (record.type == 'attributes') {
+          const changedImage = record.target;
+          const changedAttribute = record.attributeName;
+          const newCiSrcValue = changedImage.getAttribute(changedAttribute);
+          const lowPreviewDiv = changedImage.previousElementSibling;
+          const newImage = new Image();
+
+          wrapper.removeChild(changedImage);
+          wrapper.removeChild(lowPreviewDiv);
+
+          newImage.setAttribute('ci-src', newCiSrcValue);
```

Comment:
we should not hardcode ci-src as it may change by using imageSelector (mentioned before)